test/test_dwarf_expression_opcodes.py

```diff
@@ -0,0 +1,59 @@
+#------------------------------------------------------------------------------
+# elftools tests
```

Comment:
Unit tests are good, but I'm also worried about the proliferation of these small obscure `.o` files with extracted sections, which I don't know how to reproduce.

One of the keys to the library's stability is readelf-comparison tests. It would be much preferred if one could find a relevant readelf-comparison test for new added features.

Reply:
Extracted sections was another PR :) That's a discussion for elsewhere. This one has .o files, perfectly ELF. What's wrong with them?